modules/task/backend.go

```diff
@@ -118,7 +93,28 @@ func (b *inMemBackend) Name() string {
 // Start implements the Module interface
 func (b *inMemBackend) Start(ready chan<- struct{}) <-chan error {
 	b.isRunning = true
-	return make(chan error)
+	errorCh := make(chan error, 2)
```

Comment:
Since we control backend behavior in the task module, this is ok. We will not start more than once. It's ok if tests are calling it twice.